requirements/base.in

```diff
@@ -1,5 +1,6 @@
 attrs
 awesome-slugify
+base58
```

Comment:
what is this used for?

Reply:
`base58` originally came as a sub-dependency of `substrate-interface`. At some point, `substrate-interface` has been removed from the requirements, but `base58` was (and is) still in use. I am therefore adding it back as an independent package.
It is used as part of the blockchain interface in the `address_to_hex` function here: https://github.com/gridsingularity/d3a/blob/6f3505d2c9bbaaa6b6fa999cdb651f242f73850d/src/d3a/models/market/blockchain_utils.py#L81